replaced latin abbreviations with english phrases (#8861)
wagtail/wagtail: docs/extending/customising_group_views.md

{% endblock extra_fields %}
```

> Finally we configure the `wagtail.users` application to use the custom viewset, by setting up a custom `AppConfig` class. Within your project folder (that is the package containing the top-level settings and urls modules), create `apps.py` (if it does not exist already) and add:

Reviewer: ```suggestion
Finally we configure the `wagtail.users` application to use the custom viewset, by setting up a custom `AppConfig` class. Within your project folder (which will be the package containing the top-level settings and urls modules), create `apps.py` (if it does not exist already) and add:
```
